src/main/java/com/ft/jerseyhttpwrapper/AttemptLogger.java

```diff
@@ -73,7 +72,7 @@ public void stop(ResilientClient client, ClientResponse response) {
   }
 
   private void withField(IntermediateYield yield, String key, Object value) {
-    if (isBlank(key) || isNull(value) || isBlank(valueOf(value))) {
+    if (isNull(key) || isNull(value) || isNull(valueOf(value))) {
```

Comment:
Changing `isBlank` to `isNull` means that we now allow for empty strings to be logged. Is this desired behavior?

Reply:
it now uses _isBlank_ again